src/goose/brokers/redis/console/pages/home.clj

```diff
@@ -1,16 +1,16 @@
 (ns goose.brokers.redis.console.pages.home
   (:require [goose.brokers.redis.console.data :as data]
-            [goose.console :as console]
             [goose.brokers.redis.console.pages.components :as c]
+            [goose.console :as console]
             [ring.util.response :as response]))
 
 (defn- stats-bar [{:keys [prefix-route] :as page-data}]
   [:main
    [:section.statistics
     (for [{:keys [id label route]} [{:id :enqueued :label "Enqueued" :route "/enqueued"}
-                                    {:id :scheduled :label "Scheduled" :route "/scheduled"}
-                                    {:id :periodic :label "Periodic" :route "/periodic"}
-                                    {:id :dead :label "Dead" :route "/dead"}]]
+                                    {:id :scheduled :label "Scheduled" :route "/"} ;; Routed to homepage since pages are non-existent
+                                    {:id :periodic :label "Periodic" :route "/"}
+                                    {:id :dead :label "Dead" :route "/"}]]
```

Comment:
question: 
1. Do we indicate in anyway that rmq brokers won't have these routes? 
2. Have we considered not showing the unavailable stats?

Reply:
1. Not specifically. We just don't show these routes for rmq since the API doesn't support these.
2. I think showing the stats is useful irrespective of the availability of the individual pages. Either we remove the hyperlinks and show as texts or we redirect them to homepage (as current implementation).